Assets/Scripts/Game/MobilePersonMotor.cs

```diff
@@ -54,7 +54,7 @@ public class MobilePersonMotor : MonoBehaviour
 
         // References
         DaggerfallEntityBehaviour entityBehaviour;
-        MobilePersonAsset mobileAsset;
```

Comment:
You could have made this a single line change by changing this to:
```
MobilePersonAsset mobileAsset;
public MobilePersonAsset MobileAsset { get { return mobileAsset; } set { mobileAsset = value; } }
```
The rest of the code can still use the private version

Reply:
True but with this approach you have two members doing the same thing. I can change it to your approach if the overall request is okay to you?

Reply:
They're not doing the same thing: one is private access, one is public. It's idiomatic use of C# properties

Reply:
IF `mobileAsset` is made public because something else needs access to it from the outside and there are no other reasons to keep it private, it makes no sense to have it this way. Unless private mobileAsset is doing "behind the scene" stuff imho.

With you approach I can use `mobileAsset` and `MobileAsset` to achieve the same things in the class.
Another approach would be to write `public MobileAsset {private get; set}`, this way you don't have two variables doing the same thing and allowing access from the outside. Maybe this is a compromise? :D